src/frontend/package.json

```diff
@@ -1,166 +1,27 @@
 {
-  "name": "marsha",
-  "version": "4.0.0-beta.7",
-  "description": "🐠 a FUN LTI video provider",
-  "main": "front/index.tsx",
+  "name": "common",
+  "version": "0.1.0",
+  "private": true,
+  "workspaces": [
+    "apps/*",
+    "packages/*"
+  ],
   "scripts": {
-    "build": "tsc --noEmit && webpack",
-    "build-dev": "yarn copy-iframe-resizer; yarn copy-katex-fonts; yarn build",
-    "extract-translations": "formatjs extract './**/*.ts*' --ignore ./node_modules --ignore './**/*.d.ts' --out-file i18n/frontend.json --id-interpolation-pattern '[sha512:contenthash:base64:6]' --format crowdin",
-    "compile-translations": "formatjs compile-folder --format crowdin ./i18n ./translations",
-    "copy-iframe-resizer": "cp node_modules/iframe-resizer/js/*.min.js ../backend/marsha/static/js",
-    "copy-katex-fonts": "mkdir -p ../backend/marsha/static/css/fonts && cp node_modules/katex/dist/fonts/* ../backend/marsha/static/css/fonts",
-    "lint": "tslint -c tslint.json '**/*.ts?(x)' -e 'node_modules/**/*'",
-    "lint-fix": "tslint --fix -c tslint.json '**/*.ts?(x)' -e 'node_modules/**/*'",
-    "prettier": "prettier --list-different '**/*.+(ts|tsx|json|js|jsx)' --ignore-path ../.prettierignore",
-    "prettier-write": "prettier --write '**/*.+(ts|tsx|json|js|jsx)' --ignore-path ../.prettierignore",
-    "lint-all": "yarn lint-fix && yarn prettier-write",
-    "build-sass": "sass scss/_main.scss ../backend/marsha/static/css/main.css --load-path=node_modules",
-    "build-sass-production": "sass scss/_main.scss ../backend/marsha/static/css/main.css --style=compressed --load-path=node_modules",
-    "test": "TZ=UTC jest",
-    "test:windows": "SET TZ=UTC && jest",
-    "test-summary": "TZ=UTC jest 2>&1 | grep 'FAIL' | sort -t: -u -k1,1",
-    "test-coverage": "sh -c 'TZ=UTC jest $0 --coverage --collectCoverageFrom=\"$0/**\"'",
-    "test-screenshots": "TZ=UTC jest --testNamePattern='\\[screenshot\\]'"
+    "lint": "yarn workspace lib-common run build; yarn workspace lib-tests run build; yarn workspaces run lint",
+    "test": "yarn workspace lib-common run build; yarn workspace lib-tests run build; yarn workspaces run test",
+    "build-tests": "yarn workspace lib-common run build; yarn workspace lib-tests run build",
+    "build-lti": "yarn workspace lib-common run build; yarn workspace lib-components run build; yarn workspace marsha run build",
```

Comment:
@jbpenrath Didn't we made something on verna to avoid always building everything ?

Reply:
We create this command to check if a build is needed :
```sh
if [ ! -d dist ] || [ -n \"$(find src -type f -newer dist)\" ] ; then yarn build ; fi
```

But in your case, I wonder building `lib-common` is really necessary. You could import directly stuff from `src` folder then compile this code within package using it nope ?